Makefile

```diff
@@ -0,0 +1,32 @@
+.PHONY: all build rubocop test run console
+
+all: build rubocop test run
+
+ifeq ($(OS),Windows_NT)
+	detectedOS := Windows
+else
+	detectedOS := $(shell sh -c 'uname -s 2>/dev/null || echo not')
+endif
+
+docker-compose := docker-compose
+ifeq ($(detectedOS),Linux)
+	docker-compose := sudo docker-compose
```

Comment:
This doesn't seem to actually work in my VM.  I confirmed that `uname -s 2>/dev/null || echo not` does indeed return `Linux`, but it doesn't preface any docker commands with sudo.

Reply:
lol, turns out Makefiles need spaces for indenting up here, and tabs for recipes down below. Tabs up here don't work. Should be fixed now.

Reply:
Confirming that this is working correctly now :+1: